src/optick.h

```diff
@@ -83,7 +83,7 @@
 #endif
 
 // Vulkan Forward Declarations
-#ifndef OPTICK_ENABLE_GPU_VULKAN
+#if OPTICK_ENABLE_GPU_VULKAN == 0

```

Comment:
It would be better to keep "#ifndef" here is it's possible. This change enforces to have explicit define for this macro within the build system, which will complicate integration with other custom build systems. Basically for OPTICK_ENABLE_GPU_VULKAN we want two states:
1. Default - undefined
2. Enabled - defined
Keeping it this way won't break backward compatibility with all the the exising itegrations with custom build systems.